constants/appLinks.ts

```diff
@@ -5,6 +5,7 @@ type SegmentData = {
 /**
  * Represent a navigation link for the social media community links of the site
  */
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
We are not using this interface anymore, so please, remove it instead of adding the es-lint comment

Reply:
Perfect! As I am not very familiar with the project, I preferred not to delete anything.

Reply:
No worries! For me it is the same and I'm familiar with it, but...

<img width="400" alt="Screen Shot 2020-11-26 at 16 59 37" src="https://user-images.githubusercontent.com/17231966/100371910-c7822980-3008-11eb-861d-d1443ad78622.png">